src/main/java/Duke/DukeFunction/DukeList.java

```diff
@@ -0,0 +1,137 @@
+package Duke.DukeFunction;
+
+import Duke.DukeFunction.DukePrinter;
+import Duke.DukeTask.DukeDeadline;
+import Duke.DukeTask.DukeEvent;
+import Duke.DukeTask.DukeTask;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.FileWriter;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class DukeList {
+    private static final ArrayList<DukeTask> taskList = new ArrayList<>();
+    private static final String FILE_PATH = "data/list.txt";
+    private static final int MAX_TASK = 100;
```

Comment:
In the project constraints, there is no indication of a maximum size. Please do avoid limiting your `taskList` to a size if possible :)